docs/UserGuide.md

```diff
@@ -1,16 +1,148 @@
+
+#### -r, --recurse
+
+The `-r` or `--recurse` argument will recurse into child directories for each file specifier passed on the command-line. If the argument does not appear on the command-line, each file specifier will be resolved against the provided directory, if there is one, or the current working directory, if there is not.
+
+#### -c, --config
+
+The `-c` or `--config` argument can be used to pass settings, rendered as XML, that can be used to reconfigure analysis.  Accepts a single argument that specifies the path of the configuration file. See the `exportConfig` command for information on generating a preliminary configuration file that can be modified and passed back into BinSkim to reconfigure analysis.
+
+#### -q, --quiet
+
+The `-q` or `--quiet` argument suppresses BinSkim console output. BinSkim will raise an error when the -q is specified without providing a log file location to persist result via the `-o` argument.
+
+#### -s, --statistics
+
+The `-s` or `--statistics` argument configures BinSkim to record and report on various statistics related to the analysis, including total time-of-execution, the number of valid analysis targets the command-line run resolved to, and the number of invalid targets (i.e., non-portable executable files) specified for analysis.
+
+#### -h, --hashes
+
+The `-h` or `--hashes` argument configures BinSkim to emit MD5, SHA1 and SHA256 hashes of analysis targets to the SARIF output log specified via the `-o` argument. BinSkim will raise an exception in cases when -h is specified but no SARIF results file is configured via the `--output` argument. File hashes are emitted to log files to assist in results caching as well as scenarios where it is helpful to verify for auditing, compliance or other purposes that a specific version of a file was analyzed.
+
+#### -e, --environment
+
+The `-e` or `--environment` argument configures BinSkim to emit machine environment details to the SARIF log file specified via the -o argument. This information includes the user account, machine name, working directory and complete set of environment variables and definitions that were present during the analysis run. This information may be useful in some tool troubleshooting scenarios. WARNING: the information emitted by the -e argument may represent unwanted information disclosure.
+
+#### -p, --plug-in
+
+The `-p` or `--plug-in` argument is used to provide a path to a BinSkim plug-in that will be loaded and invoked at analysis time, in addition to the built-in checks. This argument can be specified multiple times on the command-line.
+
+## BinSkim Release History
+
+The latest version is always available on the main at **[NuGet](https://www.nuget.org/packages/Microsoft.CodeAnalysis.BinSkim/)**
+
+| Version | Details |
```

Comment:
> Version | Detai [](start = 1, length = 16)

We should drop the version info which is not up to date. 

We could refer to ReleaseHistory.md instead, which has this info (this information probably better suited to readme.md)